lib/simplecov/defaults.rb

```diff
@@ -29,6 +29,7 @@
   add_filter "/db/"
 
   add_group "Controllers", "app/controllers"
+  add_group "Channels", "app/channels"
```

Comment:
Maybe just check for ActionCable. I know that in Rails 5 APIs, ActionCable isn't exactly used much :p